src/Compilers/CSharp/Test/Symbol/Symbols/Source/EventTests.cs

```diff
@@ -325,6 +327,119 @@ class C
             Assert.True(fevent.RemoveMethod.IsImplicitlyDeclared, "FieldLikeEvent RemoveAccessor");
         }
 
+        [Fact]
```

Comment:
Please add a WorkItem attribute with the issue number and its complete URL. Same for other new tests.